sqlglot/dialects/postgres.py

```diff
@@ -492,7 +493,7 @@ class Generator(generator.Generator):
                 if isinstance(seq_get(e.expressions, 0), exp.Select)
                 else f"{self.normalize_func('ARRAY')}[{self.expressions(e, flat=True)}]"
             ),
-            exp.ArrayConcat: rename_func("ARRAY_CAT"),
+            exp.ArrayConcat: lambda self, e: super().arrayconcat_sql(e, name="ARRAY_CAT"),
```

Comment:
Should this be `self.arrayconcat_sql`? There's no override in postgres right? Same for the other similar entries.

Reply:
Yeah the overrides are gone now so `self <-> super` here, will revert